.buildkite/it/run.sh

```diff
@@ -4,6 +4,11 @@ set -eo pipefail
 
 source .buildkite/retry.sh
 
+function upload_logs {
+    echo "--- Upload artifacts"
+    buildkite-agent artifact upload "${RALLY_HOME}/.rally/logs/*.log"
+}
+
```

Comment:
Should this go somewhere common, like `.buildkite/retry.sh`? It relies on `$RALLY_HOME` being set, so it _could_ be fragile, is why I didn't just go ahead and do it.